client/lib/products-values/is-gsuite.js

```diff
@@ -17,6 +17,25 @@ export function isGoogleWorkspace( product ) {
 	return isGoogleWorkspaceProductSlug( product.product_slug );
 }
 
+/**
+ * Determines whether the provided Google Workspace product is for a user purchasing extra licenses (versus a new account).
+ *
+ * @param {object} product - product object
+ * @returns {boolean} - true if this product is for extra licenses, false otherwise
+ * @see isGoogleWorkspaceExtraLicence() in client/lib/purchases for a function that works on a purchase object
+ */
+export function isGoogleWorkspaceExtraLicence( product ) {
+	product = formatProduct( product );
+	assertValidProduct( product );
+
+	if ( ! isGoogleWorkspaceProductSlug( product.product_slug ) ) {
+		return false;
+	}
+
+	// Checks if that property exists has it should only be available for extra licenses
```

Comment:
Maybe be more explicit here? The sentence as written is not very clear.
e.g.
```suggestion
	// Checks if the `extra.new_quantity` property exists as it should only be specified for extra licenses
```

Reply:
I agree this could be better, thanks for the suggestion. I'll update those comments.